yarn.lock

```diff
@@ -2585,15 +2585,8 @@
     glob-to-regexp "^0.3.0"
 
 "@nicaea/types@link:types":
-  version "0.11.0"
-  dependencies:
-    "@polkadot/keyring" "^1.7.0-beta.5"
-    "@polkadot/types" "^0.96.1"
-    "@types/lodash" "^4.14.157"
-    "@types/vfile" "^4.0.0"
-    ajv "^6.11.0"
-    lodash "^4.17.15"
-    moment "^2.24.0"
+  version "0.0.0"
```

Comment:
I'm wondering what happened here

Reply:
I agree this looks quite odd. I did have to do some merge conflict resolution on yarn.lock but it wasn't at this line number. It was here: https://github.com/Joystream/joystream/pull/988/files#diff-8ee2343978836a779dc9f8d6b794c3b2R1831 but seems related.

Let me get rid of this section, re-run yarn install and see if all goes well:

```
 "@nicaea/types@link:types":
   version "0.0.0"
   uid ""
```